servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -3814,10 +1412,6 @@ void RendererSceneRenderRD::init() {
 	screen_space_roughness_limiter_limit = GLOBAL_GET("rendering/anti_aliasing/screen_space_roughness_limiter/limit");
 	glow_bicubic_upscale = int(GLOBAL_GET("rendering/environment/glow/upscale_mode")) > 0;
 	glow_high_quality = GLOBAL_GET("rendering/environment/glow/use_high_quality");
-	ssr_roughness_quality = RS::EnvironmentSSRRoughnessQuality(int(GLOBAL_GET("rendering/environment/screen_space_reflection/roughness_quality")));
```

Comment:
You missed moving ``environment_set_ssao_quality()`` its just 6 lines up, but github won't let me comment on the line directly

Reply:
Also missed `environment_set_ssil_quality` :)

Reply:
I'm wondering if we should write them out?

Reply:
I don't have a preference either way